src/main/java/me/jellysquid/mods/sodium/client/model/vertex/VanillaVertexTypes.java

```diff
@@ -8,11 +8,14 @@
 import me.jellysquid.mods.sodium.client.model.vertex.formats.particle.ParticleVertexType;
 import me.jellysquid.mods.sodium.client.model.vertex.formats.quad.QuadVertexSink;
 import me.jellysquid.mods.sodium.client.model.vertex.formats.quad.QuadVertexType;
+import me.jellysquid.mods.sodium.client.model.vertex.formats.screen_quad.ScreenQuadVertexSink;
+import me.jellysquid.mods.sodium.client.model.vertex.formats.screen_quad.ScreenQuadVertexType;
 import me.jellysquid.mods.sodium.client.model.vertex.type.VanillaVertexType;
 
 public class VanillaVertexTypes {
     public static final VanillaVertexType<QuadVertexSink> QUADS = new QuadVertexType();
     public static final VanillaVertexType<LineVertexSink> LINES = new LineVertexType();
     public static final VanillaVertexType<GlyphVertexSink> GLYPHS = new GlyphVertexType();
     public static final VanillaVertexType<ParticleVertexSink> PARTICLES = new ParticleVertexType();
+    public static final VanillaVertexType<ScreenQuadVertexSink> SCREEN_QUADS = new ScreenQuadVertexType();
```

Comment:
Many different "quad" formats are used for UI rendering, some which include light and textures. A better name for the format would be `UntexturedScreenQuad` or `BasicScreenQuad`... etc.

Reply:
This can be `BasicScreenQuad`, then a version with textures can be `TexturedScreenQuad`.
If you want `TexturedScreenQuad` with lighting, `GlyphVertexSink` (since I'm pretty sure that's the only UI-related type with a light component).